app/assets/stylesheets/application.tailwind.css

```diff
@@ -360,3 +360,34 @@
     @apply shadow-lg;
   }
 }
+
+/* Pagy */
+.pagy-nav.pagination {
+  @apply inline-flex;
+}
+
+.page {
+  @apply text-sm font-medium
+  px-3 py-1
+  border border-neutral-300/50 dark:border-neutral-750
+  hover:bg-neutral-100 dark:hover:bg-neutral-900;
+}
+
+.page.disabled {
+  @apply text-neutral-300 dark:text-neutral-800
```

Comment:
I think we need to make the disabled one of preview ones a bit brighter. it's too faint otherwise. 

match the disabled input field styling?

Reply:
we might also want to style it a bit more to be around this UI where the subtle shadows and things makes a big difference. it needs to match the input fields that we have that have the same borders, etc.

https://tailwindui.com/components/application-ui/navigation/pagination#component-69eb9381f977800aa890ce8f7d9e2d20

Reply:
![image](https://github.com/danielpaul/RapidRails/assets/40029539/af3599b2-03df-4459-ae8d-4e390de96b3c)
![image](https://github.com/danielpaul/RapidRails/assets/40029539/064eece8-ddfe-4014-b285-4245fc5cc720)